intro-2018-07.md

```diff
@@ -0,0 +1,162 @@
+
+![left](assets/stickers - joshparnham_2018-Feb-08.jpg)
+
+# Melbourne CocoaHeads
+## July 12, 2018
+### Join slack @ slack.melbournecocoaheads.com
+### Twitter @melbournecocoa
+
+---
+
+![left](assets/stickers - joshparnham_2018-Feb-08.jpg)
+
+# Melbourne CocoaHeads
+## Kicking off @ 6:30pm
+### Join slack @ slack.melbournecocoaheads.com
+### Twitter @melbournecocoa
+### Stickers up front!
+---
+
+# Welcome
+# :wave:
+
+---
+
+# Welcome :wave:
+
+![right 60%](assets/CocoaHeads.png)
+
+- "Who's this guy?"
+- Upcoming CocoaHeads events
+- Code of Conduct
+- Tonight's agenda & sponsors
+- Who's hiring
+
+---
+
+# Hi, I'm Jesse
+
+![left](assets/jtanang_2015-Aug-13.jpg)
+
+- Developing for iOS since 2010
+- realestate.com.au, Beanhunter, RMIT
+- Melbourne CocoaHeads organiser since ~2014
+
+---
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - July 18th @ Cognizant
+- Drinks Night - July 31st
+- NS Breakfast - August 3rd
```

Comment:
DevWorld? Might be good to mention it.

Reply:
I have a section in the AWS group intro now about related conferences and groups. Its gotten good feedback so far